src/framework/uicomponents/qml/Muse/UiComponents/fuzzysortfilterproxymodel.cpp

```
@@ -0,0 +1,189 @@
+    m_fuzzyPattern = simplifiedPattern;
+    compilePattern();
+
+    emit fuzzyPatternChanged(m_fuzzyPattern);
+}
+
+QString FuzzySortFilterProxyModel::filterRoleName() const
+{
+    return m_filterRoleName;
+}
+
+void FuzzySortFilterProxyModel::setFilterRoleName(const QString roleName)
```

Cppcheck (2.20.0)

[style] The function 'setFilterRoleName' is never used.

(unusedFunction)